src/Reference.php

```diff
@@ -100,59 +100,81 @@ public function getDesiredName(): string
         return '#ref_' . $this->link;
     }
 
+    public function getOurModel(): Model
+    {
+        return $this->owner;
+    }
+
+    /**
+     * @deprecated use getTheirModel instead - will be removed in dec-2020
+     */
+    public function getModel($defaults = []): Model
```

Comment:
makes sense if this was returning `getTheirModel()`

Reply:
Yes, it was